addons/l10n_tr_nilvera_einvoice_extended/views/l10n_tr_res_tax_office_views.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="l10n_tr_res_tax_office_view_list" model="ir.ui.view">
+        <field name="name">l10n_tr.res.tax.office.list</field>
+        <field name="model">l10n_tr.res.tax.office</field>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <list string="Tax Offices" multi_edit="1">
+                <field name="code" optional="hide"/>
+                <field name="name"/>
+                <field name="state_code"/>
+                <field name="state_id"/>
+            </list>
+        </field>
+    </record>
+
+    <record id="l10n_tr_res_tax_office_view_form" model="ir.ui.view">
+        <field name="name">l10n_tr.res.tax.office.form</field>
+        <field name="model">l10n_tr.res.tax.office</field>
+        <field name="mode">primary</field>
```

Comment:
Same comment than in the other file (no use of primary in a non-inheriting view)